cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -0,0 +1,229 @@
+# Introduction
+This quick start guide is meant as a guide for the very first steps to get going with DELPHI software and data access.
+
+## Overview
+
+The DELPHI stack consists of the following modules:
+
+* DELPHI DST analysis framework, also referred to as dstana. Please take a look at the [Skelana framework](docs/27002) for detailed information.
+* MonteCarlo production, consisting of simulation, reconstruction and short DST production. Note that the DST format depends on the year. During LEP1, mainly short DST was used, while during LEP2 the extendent short DST format was used. The various formats are described [here](docs/27004)
+* Event reconstruction from raw data, using the DELPHI event server.
+* The graphical DELPHI Event display, also referred to as [delgra](docs/27003).
+
+## Before you start ...
+
+Please read and accept the data access rules.
+
+DELPHI data access rules are available [here](/record/417), or from [the DELPHI web pages](https://delphi-www.web.cern.ch/delphi-www/delsec/finalrules/DELPHI_Data_preservation-8.pdf).
+
+Please read these before accessing the DELPHI software and data.
+
+## Accessing the software stack
+There are two possible ways to access the software stack.
+
+### Docker
```

Comment:
You may want to put `####` so that this would be a subsection of Accessing the software stack.

Reply:
It should already be, no ? Since "Accessing the software stack has only 2 hashes".